eng/Npm.Workspace.nodeproj

```diff
@@ -53,9 +53,9 @@
     <Message Text="Computed package version info: %(_NodePackageNameAndVersions.PackageName)=%(_NodePackageNameAndVersions.PackageVersion)" Importance="high" />
     -->
     <ItemGroup>
-      <_NodePackageNameAndVersions>
+      <_ResolvedPackageVersionInfo>
         <PackageVersion>$(PackageVersion)</PackageVersion>
-      </_NodePackageNameAndVersions>
+      </_ResolvedPackageVersionInfo>
```

Comment:
This is incorrect, isn't it? 
https://github.com/dotnet/aspnetcore/pull/53152/files#diff-87ba8ac7845060f8e3f14f5fb637b6790e56ce05e5503d2b96293301339d3910L38

returns the correct item group.

Reply:
The other part is that this doesn't need to match `<Output TaskParameter="TargetOutputs" ItemName="_ResolvedPackageVersionInfo" />` The TargetOuput here is provided in the `Returns`, `ItemName` is only where in the calling context the output needs to be placed, but they both don't have to match.

Reply:
Alright, then we must just not be setting PackageVersion in the node project anymore. There's no strong reason to use that project as far as I know, I'll try using dotnet-user-jwts.